lib/KLabeledIcon.vue

```diff
@@ -108,7 +108,7 @@
 
   .labeled-icon-wrapper {
     position: relative;
-    display: inline-block;
+    display: block;
```

Comment:
This _might_ be okay since we tend to use labeled icons as block-level elements, but I don't see any reason why this would always be the case. For example, if we decide to have a horizontal list of links, etc.